### PR TITLE
Refactor project health

### PR DESCRIPTION
**Pull request by Tymek**
## About the changes
- [x] minor updates to table style consistency
- [x] fix titles on project subpages
- [x] refactor paths
- [x] optimize project health table performance
- [x] refactor virtualized table into common component

### CODE REVIEW

**Comment by Tymek** [comment]
[Fix performance of project->health report table](https://trello.com/c/VkihrItx/1291-fix-performance-of-project-health-report-table)

File: src/component/project/Project/ProjectHealth/ReportTable/ReportStatusCell/formatStatus.ts [hunk 1]
```diff
@@ -1,5 +1,5 @@
 import { IFeatureToggleListItem } from 'interfaces/featureToggle';
-import { getDiffInDays, expired } from 'component/Reporting/utils';
+import { getDiffInDays, expired } from '../utils';
```
Comment (nunogois) [inline src/component/project/Project/ProjectHealth/ReportTable/ReportStatusCell/formatStatus.ts:2]: Do we prefer a relative path in this case?

**Review by nunogois — APPROVED** [review]
LGTM - Should we use this PR to also make all of the table header titles consistent `(X of Y)`?
Is there any table remaining that should be migrated to the new `VirtualizedTable` component that needs to be added to the backlog?

File: src/component/project/Project/ProjectHealth/ReportTable/ReportStatusCell/formatStatus.ts (hunk 1 above)
Comment (Tymek) [reply src/component/project/Project/ProjectHealth/ReportTable/ReportStatusCell/formatStatus.ts:2]: This `utils` is related only to reporting, so I thing it's OK

**Comment by Tymek** [comment]
All tables that require it right now are migrated to `VirtualizedTable`. I'll fix headers in next PR

**Pull request merged**